docker-compose.yml

```diff
@@ -37,6 +37,7 @@ services:
             - PYTHONDONTWRITEBYTECODE=1
             - RGW
             - RGW_MULTISITE=${RGW_MULTISITE:-0}
+            - EXPORTER=${EXPORTER:-0}
```

Comment:
You should probably add this to the .env.sample as well, to let users know that this option exists.